mkt/commonplace/views.py

```diff
@@ -116,12 +116,12 @@ def commonplace(request, repo, **kwargs):
     # need it.
     fxa_auth_state, fxa_auth_url = fxa_auth_info()
     site_settings = {
+        'dev_pay_providers': settings.DEV_PAY_PROVIDERS,
```

Comment:
We only need this for the site, right? Because if we want it for package, we'll want it in the site_config endpoint also.